modules/sugar/src/main/ts/ephox/sugar/api/search/Traverse.ts

```diff
@@ -19,8 +19,11 @@ const documentOrOwner = (dos: SugarElement<Node>): SugarElement<Document> =>
 
 const documentElement = (element: SugarElement<Node>) => SugarElement.fromDom(documentOrOwner(element).dom.documentElement);
 
-// The window element associated with the element
-const defaultView = (element: SugarElement<Node>) => SugarElement.fromDom(documentOrOwner(element).dom.defaultView);
+/**
+ * The window element associated with the element
+ * NOTE: this will throw if the defaultView is null.
+ */
+const defaultView = (element: SugarElement<Node>) => SugarElement.fromDom(documentOrOwner(element).dom.defaultView as Window);
```

Comment:
As above with the `!` operator

Reply:
No, that's disallowed as per our linting rules. We do this as it's more visible.